content/blog/2023-11-30-announcing-epel-support-in-elevate.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Bringing EPEL support to the ELevate project"
+type: blog
+author: 
+ name: "Andrew Lukoshko"
+ bio: "Release Engineering Lead"
+ image: /users/alukoshko.jpg
+date: '2023-11-30'
+images:
+  - /blog-images/new-repos-announcement.png
+post:
+    title: "Bringing EPEL support to the ELevate project"
+    image: /blog-images/2023-11-30-epel-elevate.jpg
+---
+
+Hello, Community! Today, AlmaLinux OS Foundation is eager to share some big news concerning  our [ELevate project](https://almalinux.org/elevate/), an initiative to support migrations between major versions of RHEL-derivatives. 
+
+Previously, the project only provided support for official operating systems repositories, excluding external repositories. However, the AlmaLinux Team made significant improvements. We are thrilled to announce that EPEL support is now included in the ELevate project, enabling smooth migrations between CentOS7 and EL8 derivatives. Please be aware that at present, there is no EPEL support available for migrating between EL8 and EL9 derivatives.
```

Comment:
```suggestion
Previously, the project only provided support for official operating systems repositories, excluding external repositories. However, the AlmaLinux Team made significant improvements. 

We are thrilled to announce that the first step in EPEL support is now included in the ELevate project, enabling smooth migrations between CentOS7 and AlmaLinux 8. 

Please be aware that at present, the EPEL support does not include migrating to non-AlmaLinux OSes, or between EL8 and EL9 derivatives, but those are planned for the future!
```

Reply:
When I talked to Andrew about this yesterday he said that it only supported AlmaLinux for now, so I wanted to clarify that in this blog post. If I misunderstood, let me know!